recipes/cron.rb

```diff
@@ -9,10 +9,16 @@
   mode 0755
 end
 
-cron_d node['certbot']['cron_name'] do
-  command '/usr/local/sbin/certbot-renew.sh'
-  user 'root'
-  (node['certbot']['cron'] || node['certbot']['default_cron']).each do |key, value|
-    send key, value
+if node['certbot']['cron_enabled']
```

Comment:
Instead of using an `if` here you could use `only_if` or `not_if` or better yet make the `action` dependent on the value of `node['certbot']['cron_enabled']` in something like this:
```ruby
action ? node['certbot']['cron_enabled'] : :create : :delete
```